EIPS/eip-2481.md

```diff
@@ -0,0 +1,166 @@
+
+**Q: Networking works today, isn't this just adding bloat?**
+
+This is adding a single integer per command while at the same time reducing code complexity and improving networking efficiency. The addition seems justified.
+
+**Q: Why not demand request ids to be sequential?**
+
+Assuming request ids start always to count from `0` upon connection, things will become messy when
+connections are lost and clients reconnect and start over with the same request ids that they had used
+in the previous session.
+
+**Q: Why allow duplicate request ids?**
+
+The main benefit is flexibility and simplicity on the implementation side. Clients could decide to share
+the same ids across multiple different request types since they are naturally separated anyway. Clients
+could even decide to not rely on request ids at all, therefore using the same constant request id across
+all requests.
+
+**Q: Why choose a 64-bit integer for the request ids**
+
+64-bit integer were chosen to keep compatibility with the `les` protocol.
+
+## Backwards Compatibility
+<!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+This EIP extends the `eth` protocol in a backwards incompatible way and requires rolling out a new version, `eth/66`. However, `devp2p` supports running multiple versions of the same wire protocol side-by-side, so rolling out `eth/66` does not require client coordination, since non-updated clients can keep using `eth/65`.
+
+This EIP does not change the consensus engine, thus does *not* require a hard fork.
+
+## Implementation
+<!--The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
+
+Trinity has a [draft PR](https://github.com/ethereum/trinity/pull/1672) with an implementation.
+
+## Security Considerations
+<!--All EIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks and can be used throughout the life cycle of the proposal. E.g. include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks and how they are being addressed. EIP submissions missing the "Security Considerations" section will be rejected. An EIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.-->
+
+None?
```

Comment:
I would it shouldn't go "final" with a question mark.

Reply:
Initially I was shy to deny security implications entirely but the EIP has had some :eyes: on it by now and nobody has brought up any concerns. I just removed the question mark now.